packages/typography/src/Typography.tsx

```diff
@@ -25,13 +25,14 @@ const h1 = css`
 
 type H1Props = HTMLElementProps<'h1'>;
 
-function H1({ children, className, ...rest }: H1Props) {
-  return (
-    <h1 {...rest} className={cx(sharedStyles, h1, className)}>
-      {children}
-    </h1>
-  );
-}
+const H1: ExtendableBox<H1Props> = ({
```

Comment:
we should pass in the default of "h1". same for other components

Reply:
Yep, good call